src/RTClib.h

```diff
@@ -423,6 +423,10 @@ class RTC_PCF8523 : RTC_I2C {
   void disableCountdownTimer(void);
   void deconfigureAllTimers(void);
   void calibrate(Pcf8523OffsetMode mode, int8_t offset);
+  int8_t readOffsetReg();
+  String getOffsetMode();  
```

Comment:
Remove trailing white space.

Reply:
Will do :) At least this should be easy.